idb/postgres/internal/encoding/encode.go

```diff
@@ -138,11 +138,24 @@ func convertLocalDeltas(deltas map[uint64]basics.StateDelta) map[uint64]stateDel
 	return res
 }
 
+func convertLogs(logs []string) [][]byte {
```

Comment:
As discussed, this needs an equivalent `unconvertLogs` function in `idb/postgres/internal/encoding/decode.go` as well.